scripts/generate_schema.py

```diff
@@ -8,36 +8,29 @@
 from collections.abc import Iterator, Sequence
 from pathlib import Path
 
+import geoarrow.pyarrow as ga
 import h5py
 import numpy as np
 import pyarrow as pa
 
 type PyArrowMetadata = dict[str | bytes, str | bytes]
 
 
-GEO_METADATA = {
-    "version": "1.1.0",
-    "primary_column": "geometry",
-    "columns": {
-        "geometry": {
-            "encoding": "point",
-            "geometry_types": ["Point"],
-            # See https://gedi.umd.edu/instrument/specifications/
-            "bbox": [-180.0, -51.6, 180.0, 51.6],
-            "covering": {
-                "bbox": {
-                    "xmax": ["bbox", "xmax"],
-                    "xmin": ["bbox", "xmin"],
-                    "ymax": ["bbox", "ymax"],
-                    "ymin": ["bbox", "ymin"],
-                }
```

Comment:
It's possible to build up the bbox as you scan through the data, i.e. as you batch-by-batch write it to GeoParquet

Reply:
I've eliminated the bbox and covering.